config/annotations.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const Joi = require('joi');
+
+// List of data, eventually key-value pairs
+const SCHEMA_ANNOTATIONS = Joi.object().optional();
```

Comment:
I can see why it would be from a code-reuse point of view, but it seems a bit odd to me that the `optional` flag is on the annotations schema itself, rather than the places where annotations are optional.